.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,56 @@
+            "openedx/core/djangoapps/geoinfo/ openedx/core/djangoapps/header_control/ openedx/core/djangoapps/heartbeat/ openedx/core/djangoapps/lang_pref/ openedx/core/djangoapps/models/ openedx/core/djangoapps/monkey_patch/ openedx/core/djangoapps/oauth_dispatch/ openedx/core/djangoapps/olx_rest_api/ openedx/core/djangoapps/password_policy/ openedx/core/djangoapps/plugin_api/ openedx/core/djangoapps/plugins/ openedx/core/djangoapps/profile_images/ openedx/core/djangoapps/programs/ openedx/core/djangoapps/safe_sessions/ openedx/core/djangoapps/schedules/ openedx/core/djangoapps/self_paced/ openedx/core/djangoapps/service_status/ openedx/core/djangoapps/session_inactivity_timeout/ openedx/core/djangoapps/signals/ openedx/core/djangoapps/site_configuration/ openedx/core/djangoapps/system_wide_roles/ openedx/core/djangoapps/theming/ openedx/core/djangoapps/user_api/ openedx/core/djangoapps/user_authn/ openedx/core/djangoapps/util/ openedx/core/djangoapps/verified_track_content/ openedx/core/djangoapps/video_config/ openedx/core/djangoapps/video_pipeline/ openedx/core/djangoapps/waffle_utils/ openedx/core/djangoapps/xblock/ openedx/core/djangoapps/xmodule_django/ openedx/core/djangoapps/zendesk_proxy/ openedx/core/djangolib/ openedx/core/lib/ openedx/core/tests/ openedx/core/tests/ openedx/features/ openedx/testing/ openedx/tests/",
+            "cms/djangoapps/api/ cms/djangoapps/cms_user_tasks/ cms/djangoapps/course_creators/ cms/djangoapps/export_course_metadata/ cms/djangoapps/maintenance/ cms/djangoapps/models/ cms/djangoapps/pipeline_js/ cms/djangoapps/xblock_config/ cms/envs/ cms/lib/",
+            "cms/djangoapps/contentstore/",
+            "common/djangoapps/",
+            "common/lib/",
+        ]
+
+
+    name: python ${{ matrix.python-version }},django ${{ matrix.django-version }} ${{ matrix.test_module }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: start mongodb service
+        run: |
+          sudo /etc/init.d/mongodb start
+
+      - name: set module name
+        run: |
+          echo "module_name=$(echo '${{ matrix.test_module }}' | awk -F '/' '{print $1}')" >> $GITHUB_ENV
+
+      - name: set settings path
+        run: |
+          echo "settings_path=$(if [ '${{ env.module_name }}' = 'cms' ]; then echo 'cms.envs.test'; else echo 'lms.envs.test' ; fi)" >> $GITHUB_ENV
+
+      - name: install requirements
+        run: |
+          sudo pip install -r requirements/pip.txt
+          sudo pip install -r requirements/edx/testing.txt
+          sudo pip install "django~=${{ matrix.django-version }}.0"
```

Comment:
How will this get managed? Would we just update if there is a bug fix that we need to pass tests?

Reply:
we're not specifying the Django version here, it is being pulled from the `matrix`, although we already have Django in requirements files, installing it this way allows us to test multiple Django versions for each matrix item in future
.
let me know if that answers your question

Reply:
Thanks. Actually, I missed `~=` and wasn't clear it would install the highest patch version, where I had thought that the 0 patch version was installed. I'm all set.